app/src/main/java/com/asfoundation/wallet/support/SupportInteractor.kt

```diff
@@ -88,4 +89,10 @@ class SupportInteractor(private val supportRepository: SupportRepository,
   fun getUnreadConversationCount() = Observable.just(Intercom.client().unreadConversationCount)
 
   private fun getUnreadConversations() = Intercom.client().unreadConversationCount
+
+  private fun levelBeingInvalidated(newLevel: Int, currentUserLevel: Int): Boolean {
+    // in case of error for same user address (newLevel=-1 and currentLevel>=0),
+    // user info isn't left in inconsistent state
+    return (newLevel < 0 && currentUserLevel > newLevel)
```

Comment:
Note that this condition is only true if the level to be inserted (newLevel) is an invalid one, e.g., -1. 
For a case like newLevel=1 and currentUserLevel=2, this condition is false (because newLevel>=0) and so it won't stop the update of the user level on intercom.
Perhaps it isn't clear what this condition means? I'll try to think of a better name. If you have any suggestions, they are always welcome!

Reply:
This method has been removed since it is no longer necessary, with the changes made in the repository